### PR TITLE
is_prime added

### PR DESCRIPTION
**Pull request by tahumada**
function to check for prime numbers added
To check:
$ ipython
In [1]: from simple_functions import *  
In [2]: is_prime(5)

### CODE REVIEW

**Comment by joriswitstok** [comment]
I will review this!

File: Ahumada/template/simple_functions.py [hunk 1]
```diff
@@ -10,3 +10,12 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(x):
```
Comment (joriswitstok) [inline Ahumada/template/simple_functions.py:14]: What if the argument provided to the function is not an integer?

File: Ahumada/template/simple_functions.py [hunk 2]
```diff
@@ -10,3 +10,12 @@ def factorial(value):
...
+    if x >= 2:
+        for y in range(2,x):
+            if not ( x % y ):
+                return False
+    else:
+        return False
+    return True
```
Comment (joriswitstok) [inline Ahumada/template/simple_functions.py:21]: Nice and compact function!

**Review by joriswitstok — COMMENTED** [review]
Seems like the new function will work well.

**Comment by astrofrog** [comment]
Thanks for opening the pull request and for the review(s)! Below I have provided a few comments about the *process* of the pull request and review. Note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code. Also, because there are pending suggestions, we will not merge this PR until the end of Astro Hack Week.

* @tahumada - related to the description, on github,code samples are best done either with the ``` `` ``` (inline) or `` ``` `` (block) - those render like this: ``some code`` or:
```
if code:
    use_this()
```
it's best to use these tools, particularly for python code where the formatting very much matters (without these, the indentation disappears).

* @joriswitstok - you did not select "approve" or "request changes" when you submitted the review.  Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments.

**Comment by bsipocz** [comment]
As a closure to AstroHackWeek, we go ahead and merge this now. Thank you both.

**Pull request merged**